app/src/main/res/layout/download_dialog_layout.xml

```diff
@@ -75,7 +75,6 @@
         android:layout_width="wrap_content"
         android:layout_marginTop="8dp"
         android:layout_marginEnd="16dp"
-        android:backgroundTint="?accent"
```

Comment:
We are removing the backgroundTint here since this is already provided by the `@style/PositiveButton`